frontend/src/bundles/common/pages/budgets/budget-details/budget-details.tsx

```diff
@@ -172,6 +101,7 @@ const BudgetDetails = (): JSX.Element => {
     useEffect(() => {
         void dispatch(budgetsActions.loadAll());
         void dispatch(categoriesActions.loadCategories());
+        void dispatch(loadTransactions());
```

Comment:
transactionActions.loadTransactions();

Reply:
fix in [https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/367/commits/41d8ea7e508c7c45c1cfa09406fa3012e4fce354](url)